### PR TITLE
Fixed casing of header on NuGet.org to comply with Windows/VS standards.

### PR DESCRIPTION
**Pull request by chgill-MSFT**
Addresses https://github.com/NuGet/NuGetGallery/issues/8008

### CODE REVIEW

File: src/NuGetGallery/Areas/Admin/Views/ApiKeys/Index.cshtml [hunk 1]
```diff
@@ -65,14 +65,14 @@
                             <code>Description:</code> <span data-bind="text: ApiKeyViewModel.Description"></span><br />
                         </span>
                         <!-- ko if: LeakedUrl -->
-                        <code>Leaked Url:</code> <a data-bind="attr: { href: LeakedUrl }">Link</a><br />
+                        <code>Leaked URL:</code> <a data-bind="attr: { href: LeakedUrl }">Link</a><br />
```
Comment (joelverhagen) [inline src/NuGetGallery/Areas/Admin/Views/ApiKeys/Index.cshtml:68]: 👌

File: src/NuGetGallery/Views/Shared/Confirm.cshtml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ﻿@model ConfirmationViewModel
 @{
     var titleString = Model.ConfirmingNewAccount ? (Model.IsOrganization ? "Organization" : "Account") : "Email";
```
Comment (joelverhagen) [inline src/NuGetGallery/Views/Shared/Confirm.cshtml:3]: These should be lowercase too right?

File: src/NuGetGallery/Areas/Admin/Views/DeleteAccount/DeleteAccountStatus.cshtml [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ﻿@using NuGetGallery
 @model NuGetGallery.Areas.Admin.ViewModels.DeleteAccountStatus
 @{
-    ViewBag.Title = "Delete Account Status" + Model.AccountName;
+    ViewBag.Title = "Delete account status" + Model.AccountName;
```
Comment (loic-sharma) [inline src/NuGetGallery/Areas/Admin/Views/DeleteAccount/DeleteAccountStatus.cshtml:4]: Is this missing a space?

**Comment by loic-sharma** [comment]
> Given our minimal test coverage of web pages I think it would be prudent to do a manual pass of all affected pages to make sure there's no problems.

+1. We should also check for pages that may have the incorrect capitalization

File: src/NuGetGallery/Views/Users/_UserPackagesListForDeletedAccount.cshtml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿@model DeleteAccountViewModel
 @{
-    var isOrganization = Model.User is Organization;
+    var isOrganization = Model.User is organization;
```
Comment (zhhyu) [inline src/NuGetGallery/Views/Users/_UserPackagesListForDeletedAccount.cshtml:3]: This is not for UI but for type check. We should use the upper case here, which denotes the "Organization" class.

Comment (loic-sharma) [inline src/NuGetGallery/Views/Users/_UserPackagesListForDeletedAccount.cshtml:3]: This change should be reverted (it's what caused the CI to fail)

Comment (zhhyu) [reply src/NuGetGallery/Views/Users/_UserPackagesListForDeletedAccount.cshtml:3]: I am running before you. :smile:

Comment (loic-sharma) [reply src/NuGetGallery/Views/Users/_UserPackagesListForDeletedAccount.cshtml:3]: You beat me by seconds! 🤣

**Comment by drewgillies** [comment]
This seems to have gone stale. Should we rebase/work toward getting this merged? @chgill-MSFT @skofman1 @joelverhagen @zhhyu @agr @loic-sharma

**Comment by drewgillies** [comment]
This still hasn't progressed. What are your thoughts on closing this vs completing it? @chgill-MSFT @skofman1 @joelverhagen @zhhyu @agr @loic-sharma

**Comment by joelverhagen** [comment]
Let's close it. We'll take this later if it needs attention.

**Pull request closed without merging**